includes/html/pages/device/apps/smart.inc.php

```diff
@@ -20,16 +20,22 @@
 foreach ($app_data['disks'] as $label => $disk_data) {
     $disk = $label;
 
-    if ($vars['disk'] == $disk) {
+    if (array_key_exists('disk', $vars) && ($vars['disk'] == $disk)) {
         $label = '<span class="pagemenu-selected">' . $label . '</span>';
     }
 
     if (isset($app_data['disks'][$disk]['health_pass'])) {
         if ($app_data['disks'][$disk]['health_pass'] == 1) {
-            $health_status = '(OK)';
+            $health_status_text = '(OK)';
+            $health_status_color = 'green';
         } else {
-            $health_status = '(FAIL)';
+            $health_status_text = '(FAIL)';
+            $health_status_color = 'red';
         }
+
+        $health_status = '<font color="' . $health_status_color . '">' . $health_status_text . '</font>';
```

Comment:
font tag is deprecated. use css, bootstrap, or tailwind instead.

again, init $health_status outside if.

Reply:
changed, 
but health_status has to be defined in loop because of potential different status of disks

Reply:
yep, start of the loop was correct.